scripts/core/auth/login-modal-directive.ts

```diff
@@ -16,7 +16,7 @@ angular.module('superdesk.core.auth.login', []).directive('sdLoginModal', [
     function(session, auth, features, config, deployConfig, usersService, notify, $route) {
         return {
             replace: true,
-            template: require('./login-modal.html'),
+            templateUrl: 'scripts/core/auth/login-modal.html',
```

Comment:
I'd like the value to be received from our extensibility/plugin system, but since we don't have one yet, I think it's best to add this to `superdesk.config.js`.

Reply:
What would you put inside config? The path to the template?

Reply:
yep. `loginTemplateUrl: 'scripts/core/auth/login-modal.html'`

Reply:
I don't know if it would be a good idea to start declaring template paths on config. And if it is I would rather not put it in the core. just on client repos. So it would be `templateUrl: config.templateUrl || 'scripts/core...'`

Reply:
As we don't have an extension system, I don't see a better place than the config. It makes it obvious that this part is extensible which wouldn't be clear otherwise.

Reply:
makes sense to make it explicit

Reply:
@petrjasek in the code or in the config file?

Anyway, I tried the config option and it's not working because even if I put the path there, this directive won't find the path on the outter repo

Reply:
If it doesn't find the path, you can pass the entire template string instead by using `template` instead `templateUrl` in the directive.

Reply:
@tomaskikutis template doesn't find the path either as it needs a `require()`

Reply:
How about using `require` in the config?
`loginTemplate: require('scripts/core/auth/login-modal.html')`

Reply:
@tomaskikutis doesn't work because Grunt fails to start: `SyntaxError: Unexpected token <` it's probably trying to load the html as a module and of course it fails

Reply:
Keep it as is then and add a comment that the path is overwritten in fi repo.